minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloCluster.java

```diff
@@ -109,6 +109,14 @@ public void stop() throws IOException, InterruptedException {
     impl.stop();
   }
 
+  /**
+   * @since 2.0.1
+   */
+  @Override
+  public void close() throws IOException, InterruptedException {
+    this.stop();
+  }
+
```

Comment:
It would probably be better to not throw InterruptedException, but to catch it, and reset the current thread's interrupt status. I found this good explanation of how to handle these: https://dzone.com/articles/how-to-handle-the-interruptedexception

Reply:
The most recent commit contains my attempt to include this suggestion. @ctubbsii